Modified docs to adhere to ibm style guide (#57)
devfile/docs: docs/modules/user-guide/partials/proc_adding-projects-to-a-devfile.adoc

----
====

. For each project, define the optional `sparseCheckoutDir` attribute to populate the project with selected directories.
+
[NOTE]
====

> * Set your project to `/my-module/` to create only the root `my-module` directory along with its content.

Reviewer: ```suggestion
* Set the project to `/my-module/` to create only the root `my-module` directory along with its content.
```

* Omit the leading slash,`my-module/`, to create all `my-module` directories that exist in the project, including, `/addons/my-module/`.

> ** The trailing slash indicates that only directories and their content with the given name are created.

Reviewer: ```suggestion
** Add a trailing slash to create only directories with the given name (and their content).
```
